tests/zfs-tests/cmd/draid/draid.c

```diff
@@ -24,6 +24,13 @@
  */
 
 #include <stdio.h>
+#ifdef	__dilos__
+#include <errno.h>
+#include <stdlib.h>
+#include <strings.h>
+#include <sys/types.h>
+#include <fcntl.h>
+#endif
```

Comment:
Let's include these extra headers on all platforms.  They're probably being brought in indirectly on Linux and FreeBSD which is why we didn't see a build issue.